starknet_devnet/devnet_config.py

```diff
@@ -276,6 +289,12 @@ def parse_args(raw_args: List[str]):
         type=_fork_block,
         help="Specify the block number where the --fork-network is forked; defaults to latest",
     )
+    parser.add_argument(
+        "--chain-id",
+        type=_chain_id,
+        default=StarknetChainId.TESTNET.name,
+        help="Specify the chain id as string, MAINNET or TESTNET",
```

Comment:
This help string can use the same collection of legal values as the `_chain_id` validation function (extract, see DUMP_ON_OPTIONS for reference).

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/358/commits/5d47c4af1fe8a94cb6b6e799aa4100058c544f50